rpc-client-types/src/filter.rs

```diff
@@ -154,7 +154,7 @@ impl Memcmp {
         self.offset
     }
 
-    pub fn bytes(&self) -> Option<Cow<Vec<u8>>> {
+    pub fn bytes(&self) -> Option<Cow<'_, Vec<u8>>> {
```

Comment:
This should probably be marked as 'static here to convey that the CoW is owned.

Reply:
one of the branches returns `Borrowed`, so this can't be `'static`